controllers/configuration_controller.go

```diff
@@ -740,6 +714,17 @@ func (meta *TFConfigurationMeta) assembleTerraformJob(executionType TerraformExe
 	}
 	initContainers = append(initContainers, tfPreApplyInitContainer)
 
+	containerMountPointList := []v1.VolumeMount{
```

Comment:
Can you revert the separate variable or change the name to `containerVolumeMounts` or `containerVolumeMountList`? It seems not very clear.

Reply:
OK. I will revert the code.